README.md

```diff
@@ -1,37 +1,48 @@
+![GitHub Repo stars](https://img.shields.io/github/stars/synthesized-io/insight?logo=github&logoColor=%23333&style=social)
 
-Installation
---------------
+A python package to quickly **understand**, **assess**, and **compare** pandas `Series`/`DataFrames`.
 
-### 1. Install python requirements:
-#### macOS
-(make sure you have brew installed)
-```
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
-```
-```
-brew install openssl readline sqlite3 xz zlib
-```
-#### ubuntu
-```
-sudo apt-get update; sudo apt-get install make build-essential libssl-dev zlib1g-dev \
-libbz2-dev libreadline-dev libsqlite3-dev wget curl llvm \
-libncursesw5-dev xz-utils tk-dev libxml2-dev libxmlsec1-dev libffi-dev liblzma-dev
-```
+The predominant functions in the package focus on easy-to-use **metrics** and intelligent **plotting functions**. The metrics can also be configured from YAML to allow for simple to configure benchmarking and assessment scripts.
 
-### 2. Install pyenv:
-#### macOS
-```
-brew update
-brew install pyenv
-```
-#### ubuntu
-```
-curl https://pyenv.run | bash
+
+## [![PyPI](https://img.shields.io/pypi/v/insight?color=%23DA5&label=latest&logo=semver)](https://pypi.org/project/insight/) [![CodeQL Status](https://img.shields.io/github/workflow/status/synthesized-io/insight/CodeQL?color=%236ac&label=CodeQL&logo=GitHub%20Actions&logoColor=%236ac)](https://github.com/synthesized-io/insight/actions/workflows/code-ql.yml) [![CI Status](https://img.shields.io/github/workflow/status/synthesized-io/insight/CI?label=CI&logo=GitHub%20Actions&logoColor=%23afb)](https://github.com/synthesized-io/insight/actions/workflows/CI.yaml) [![Coverage](https://sonarcloud.io/api/project_badges/measure?project=synthesized-io_synthesized-insight&metric=coverage)](https://sonarcloud.io/summary/new_code?id=synthesized-io_synthesized-insight) [![Code Smells](https://sonarcloud.io/api/project_badges/measure?project=synthesized-io_synthesized-insight&metric=code_smells)](https://sonarcloud.io/summary/new_code?id=synthesized-io_synthesized-insight) [![pre-commit.ci status](https://results.pre-commit.ci/badge/github/synthesized-io/insight/master.svg)](https://results.pre-commit.ci/latest/github/synthesized-io/insight/master)
+
+
+## Installation
+
+```console
+$ pip install 'insight'
 ```
-### 3. Install python 3.8.10:
+
+## Usage
+
+### Metrics
+
+At the core of insight are the metrics classes which can be evaluated on one series, two series,
+one dataframe or two dataframes.
+
+```pycon
+>>> import insight.metrics as m
+>>> metric = m.EarthMoversDistance()
+>>> metric(df['A'], df['B'])
+0.14
 ```
-pyenv install 3.8.10
+
+### Plotting
+
+The package provides various plotting functions which allow you to easily explore any series, dataframe
+or multiple dataframes.
+
+```pycon
+>>> import insight.plotting as p
+>>> p.plot_dataset(df)
```

Comment:
As this function plots multiple datasets through the use of a list, it is probably better to have the example code instead be:
```pycon
>>> import insight.plotting as p
>>> p.plot_dataset([df_a, df_b])
```

Reply:
Ah well spotted. I'll make that change.